<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>terceira_tela</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>20799132821586787818-128.png</normaloff>20799132821586787818-128.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color: rgb(120, 116, 170);
border-radius: 15px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>401</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>50</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Cadastro</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>180</y>
         <width>411</width>
         <height>541</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="6" column="0">
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
background-color: rgb(227, 178, 100);
border-radius: 15px;
border-color: rgb(21, 30, 63);
border-style: outset;
border-width: 2px;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../icons/padlock.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <property name="indent">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;
</string>
          </property>
          <property name="placeholderText">
           <string>Digite sua rua e n√∫mero</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;
</string>
          </property>
          <property name="placeholderText">
           <string>Digite seu cpf</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;
</string>
          </property>
          <property name="placeholderText">
           <string>Digite seu email</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Senha</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
background-color: rgb(227, 178, 100);
border-radius: 15px;
border-color: rgb(21, 30, 63);
border-style: outset;
border-width: 2px;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../icons/birth.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <property name="indent">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
background-color: rgb(227, 178, 100);
border-radius: 15px;
border-color: rgb(21, 30, 63);
border-style: outset;
border-width: 2px;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../icons/email.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <property name="indent">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_6">
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;
</string>
          </property>
          <property name="placeholderText">
           <string>Digite seu whatsapp</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_1">
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Digite seu nome</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
background-color: rgb(227, 178, 100);
border-radius: 15px;
border-color: rgb(21, 30, 63);
border-style: outset;
border-width: 2px;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../icons/map.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <property name="indent">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_1">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
background-color: rgb(227, 178, 100);
border-radius: 15px;
border-color: rgb(21, 30, 63);
border-style: outset;
border-width: 2px;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../icons/13995090641543238940-128.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <property name="indent">
           <number>-5</number>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDateEdit" name="dateEdit_5">
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;
</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="displayFormat">
           <string>dd-MM-yyyy</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
background-color: rgb(227, 178, 100);
border-radius: 15px;
border-color: rgb(21, 30, 63);
border-style: outset;
border-width: 2px;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../icons/id.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <property name="indent">
           <number>-5</number>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEdit_7">
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;
</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Digite sua senha</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_8">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
background-color: rgb(227, 178, 100);
border-radius: 15px;
border-color: rgb(21, 30, 63);
border-style: outset;
border-width: 2px;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../icons/padlock.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <property name="indent">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="lineEdit_8">
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>280</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;
</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Confirme sua senha</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
background-color: rgb(227, 178, 100);
border-radius: 15px;
border-color: rgb(21, 30, 63);
border-style: outset;
border-width: 2px;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../icons/whatsa.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <property name="indent">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>770</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Registrar</string>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;
background-color: rgb(227, 178, 100);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/usercheked.png</normaloff>../icons/usercheked.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_1</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>dateEdit_5</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>lineEdit_7</tabstop>
  <tabstop>lineEdit_8</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
